app/Domains/Redis/Jobs/StoreInsightsJob.php

```diff
@@ -0,0 +1,52 @@
+<?php
+
+namespace App\Domains\Redis\Jobs;
+
+use App\Data\Collections\MediaCollection;
+use App\Data\Models\Account;
+use App\Data\Models\Insights;
+use App\Data\Models\Metrics;
+use App\Data\Repositories\InsightsRepository;
+use Lucid\Units\Job;
+use Illuminate\Support\Facades\Redis;
+
+class StoreInsightsJob extends Job
```

Comment:
it's best if you create separate methods for storing the different kinds of insights:

- account
- medias
- metrics

Each of these storage jobs would require the object that it needs to store (don't pass `message`).

Reply:
done